libwallet/assets/wallet/wallet_utils.go

```diff
@@ -161,92 +160,89 @@ func naclLoadFromPass(pass []byte) (nacl.Key, error) {
 	return nacl.Load(utils.EncodeHex(hash))
 }
 
-// encryptWalletSeed encrypts the seed with secretbox.EasySeal using pass.
-func encryptWalletSeed(pass []byte, seed string) ([]byte, error) {
+// encryptWalletMnemonic encrypts the mnemonic with secretbox.EasySeal using pass.
+func encryptWalletMnemonic(pass []byte, mnemonic string) ([]byte, error) {
 	key, err := naclLoadFromPass(pass)
 	if err != nil {
 		return nil, err
 	}
-	return secretbox.EasySeal([]byte(seed), key), nil
+	return secretbox.EasySeal([]byte(mnemonic), key), nil
 }
 
-// decryptWalletSeed decrypts the encryptedSeed with secretbox.EasyOpen using pass.
-func decryptWalletSeed(pass []byte, encryptedSeed []byte) (string, error) {
+// decryptWalletMnemonic decrypts the encryptedMnemonic with secretbox.EasyOpen using pass.
+func decryptWalletMnemonic(pass []byte, encryptedMnemonic []byte) (string, error) {
 	key, err := naclLoadFromPass(pass)
 	if err != nil {
 		return "", err
 	}
 
-	decryptedSeed, err := secretbox.EasyOpen(encryptedSeed, key)
+	decryptedMnemonic, err := secretbox.EasyOpen(encryptedMnemonic, key)
 	if err != nil {
 		return "", errors.New(utils.ErrInvalidPassphrase)
 	}
 
-	return string(decryptedSeed), nil
+	return string(decryptedMnemonic), nil
 }
 
 // For use with gomobile bind,
 // doesn't support the alternative `GenerateSeed` function because it returns more than 2 types.
-func generateSeed(assetType utils.AssetType, wordSeedType WordSeedType) (v string, err error) {
+func generateMnemonic(wordSeedType WordSeedType) (v string, err error) {
 	var entropy []byte
 	//33-word seeds and 24-word seeds both use length 32 (256 bits) while 12-word seed uses length 16 (128 bits).
 	var length uint8 = dcrhdkeychain.RecommendedSeedLen
 	if wordSeedType == WordSeed12 {
 		length = dcrhdkeychain.MinSeedBytes
 	}
-	switch assetType {
-	case utils.BTCWalletAsset:
-		entropy, err = btchdkeychain.GenerateSeed(length)
-		if err != nil {
-			return "", err
-		}
-	case utils.DCRWalletAsset:
-		entropy, err = dcrhdkeychain.GenerateSeed(length)
-		if err != nil {
-			return "", err
-		}
-	case utils.LTCWalletAsset:
-		entropy, err = ltchdkeychain.GenerateSeed(length)
-		if err != nil {
-			return "", err
-		}
+
+	entropy, err = btchdkeychain.GenerateSeed(length)
```

Comment:
i'm guessing the reason only btchdkeychain was used is because all other assets have the same underlying logic in their keychain, in that case, a comment should b left tot hat effect